01.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -940,6 +942,24 @@ This rule redirects all requests to example.org/test.mp4 to the resource named n
 
 >More information on scriptlets, redirects, and their usage is available in [this GitHub section](https://github.com/AdguardTeam/Scriptlets#redirect-resources).
 
+
+<a id="denyallow-modifier"></a>
+
+#### **`denyallow`**
+
+This modifier makes it easier to implement default-deny/allow-exceptionally in static filter lists.
```

Comment:
Tbh, I don't like how it's worded.

@sxgunchenko can we somehow rephrase this and use AdGuard terminology.